src/components/pages/Registration.tsx

```diff
@@ -115,7 +107,7 @@ export const Registration = () => {
   );
 };
 
-const ticketList: Ticket[] = [
+/*const ticketList: Ticket[] = [
```

Comment:
No commented code please. If you want to retain the code fragment then switch off this component using a flag.